tests/modules/gatk4/mutect2/main.nf

```diff
@@ -28,6 +29,28 @@ workflow test_gatk4_mutect2_tumor_normal_pair {
     GATK4_MUTECT2_PAIR ( input, fasta, fai, dict, germline_resource, germline_resource_tbi, panel_of_normals, panel_of_normals_tbi )
 }
 
+workflow test_gatk4_mutect2_tumor_normal_pair_f1r2 {
+    input = [ [ id:'test', normal_id:'normal', tumor_id:'tumour' ], // meta map
```

Comment:
why does this work here? we also didn't set any id's in the test data as far as I know

Reply:
no idea, I'm assuming at some point the bam header contains this normal or tumour, but no idea how it arrived there...

Reply:
I guess we need to sort this part out in sarek, and maybe update the module here